src/Types.jl

```diff
@@ -1026,23 +1026,38 @@ end
 
 "Determine a single UUID for a given name, prompting if needed"
 function registered_uuid(env::EnvCache, name::String)::UUID
-    uuids = registered_uuids(env, name)
+    uuids = unique(registered_uuids(env, name))
```

Comment:
The `unique` call should probably be part of the definition of `registered_uuids`.

Reply:
Done